docs/7.Scan Examples/OpenAPI.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: default
+published: true
+title: OpenAPI configuration scanning
+nav_order: 20
+---
+
+# OpenAPI configuration scanning
+Checkov supports the evaluation of policies on your OpenAPI files.
+When using checkov to scan a directory that contains a OpenAPI manifests it will validate if the file is compliant with OpenAPI best practices such as securityDefinitions and security requirement are well-defined, and more.  
```

Comment:
```suggestion
When using checkov to scan a directory that contains OpenAPI manifests it will validate if the file is compliant with OpenAPI best practices such as securityDefinitions and security requirement are well-defined, and more.  
```